aten/src/ATen/native/cuda/Normalization.cuh

```diff
@@ -781,6 +693,7 @@ void batch_norm_elemt_cuda_template(Tensor& output_, const Tensor& input_, const
   int tb = std::max<int>(64/tf, 1);
   dim3 blocks_trans(input.size(1), std::max<int>(1, std::min<int>((256*1024)/input.size(1),
                                                                   (input.size(0)+tb-1)/tb)));
+  blocks_trans.y = std::min(blocks_trans.y, MAX_GRID_SIZE);
```

Comment:
This is a bug-fix BTW. Large batch sizes with few channels result in `block_trans.y` greater than the max launch bound. This test fails, so presumably this was spotted for `BatchNorm` after the code was copied for `SyncBatchNorm`.

https://github.com/pytorch/pytorch/blob/208f81b7877f96dafa7b0b4dc680d697545c2ae3/test/test_nn.py#L8245